### ISSUE TITLE
fix css/style for dashboard becomes distorted at certain width

### ISSUE THREAD

**Issue by polymahh** [idurar/idurar-erp-crm#332]
at a certain width the dashboard becomes distorted as you can see 
![image](https://github.com/idurar/idurar-erp-crm/assets/15654652/62f96526-fcbd-4723-9b3e-73af1ff90bf0)
![image](https://github.com/idurar/idurar-erp-crm/assets/15654652/c21fc519-b926-467f-a599-693470d4f40d)
![image](https://github.com/idurar/idurar-erp-crm/assets/15654652/6a270b93-6601-49b0-92af-884481c77eea)

we need to change the css to keep it smooth and responsive even if we change the numbers or the width

**Comment by Vishrut19** [comment]
I want to work on it. Please assign me

**Comment by polymahh** [comment]
@Vishrut19 i cant assign it but please feel free to help

**Comment by Vishrut19** [comment]
Can you please specify the width a which this issue is happening?

**Comment by polymahh** [comment]
at 1060px to 994px
but we are not looking to fix at a certain width we may need to remake the cards with flexbox
and also if the number gets longer it is going off border 
![image](https://github.com/idurar/idurar-erp-crm/assets/15654652/b2ccc257-5fa6-4e15-a50e-687cf628aea6)

**Comment by Vishrut19** [comment]
> at 1060px to 994px but we are not looking to fix at a certain width we may need to remake the cards with flexbox and also if the number gets longer it is going off border ![image](https://user-images.githubusercontent.com/15654652/263246885-b2ccc257-5fa6-4e15-a50e-687cf628aea6.png)

Then I think changing the CSS properties would help I guess

**Comment by Vishrut19** [comment]
I was working on the project but I can't see these numbers as the above-mentioned screenshot  
<img width="1470" alt="Screenshot 2023-08-25 at 4 57 52 PM" src="https://github.com/idurar/idurar-erp-crm/assets/53470896/e82563bd-b2b3-493f-8042-823e0c382c0d">

**Comment by polymahh** [comment]
that's because we are using deferent databases , you can try adding numbers statically

**Comment by Vishrut19** [comment]
> that's because we are using deferent databases , you can try adding numbers statically

How to do that? Can you please suggest?

**Comment by polymahh** [comment]
i guess you can do it like this 

![image](https://github.com/idurar/idurar-erp-crm/assets/15654652/6e1fda3c-5e2f-4a11-9970-e0f5e154318f)

but if you think this is a bit harder for you there is no problem
you can leave it for @Ando22 he was working on the dashboard before

**Comment by Vishrut19** [comment]
> i guess you can do it like this
> 
> ![image](https://user-images.githubusercontent.com/15654652/263265518-6e1fda3c-5e2f-4a11-9970-e0f5e154318f.png)
> 
> but if you think this is a bit harder for you there is no problem you can leave it for @Ando22 he was working on the dashboard before

let me give it a try and if I have some problem then definitely I will let you know @polymahh

**Comment by brunovjk** [comment]
Sorry @Vishrut19 if I'm intruding on your work, but:
@polymahh I created a [draft pull request](https://github.com/idurar/idurar-erp-crm/pull/363), I made some changes to the CSS of the Dashboard components.

But Still need to deal with number gets longer it is going off border
263246885-b2ccc257-5fa6-4e15-a50e-687cf628aea6

What would be a better approach? Does it break the number into two or more columns, or into specific width using column instead of row?
